src/Stratis.SmartContracts.IntegrationTests/Stratis.SmartContracts.IntegrationTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <AssemblyName>Stratis.SmartContracts.IntegrationTests</AssemblyName>
-    <PackageId>Stratis.SmartContracts.IntegrationTests</PackageId>
```

Comment:
I'd say that anything related to SC and Sidechains we just ignore

Reply:
Agreed will update